include/cooperative_perception/clustering.hpp

```diff
@@ -0,0 +1,230 @@
+#ifndef COOPERATIVE_PERCEPTION_CLUSTERING_HPP
```

Comment:
you should know what this comment is going to be by now. truly not even that important but-

Leidos copyright header needed